pages/index.js

```diff
@@ -1,11 +1,22 @@
 import ItemList from '../components/ItemList';
 import Header from '../components/Header';
+import Footer from '../components/Footer';
+import { useState } from 'react';
+import { nanoid } from 'nanoid';
+import { dummyitems } from '../lib/data';
 
 export default function Home() {
+  const [items, setItems] = useState([...dummyitems]);
+  function handleAddItem(newItem) {
+    setItems([{ name, date, quantity, id: nanoid(), ...newItem }, ...items]);
```

Comment:
```suggestion
    setItems([newItem, ...items]);
```

Oh now i see that you added an ID already!
I think its is a little bit more structured if you add the id where you build the whole object (in AddForm.js), but as you wish. this comment is only a suggestion :)